README.md

```diff
@@ -221,7 +221,7 @@ let pool = device.create_command_pool(&pool_create_info).unwrap();
 
 ### Optional linking
 
-The default `linked` cargo feature will link your binary with the Vulkan loader directly and expose the infallible `Entry::new`.
+The default `linked` cargo feature will link your binary with the Vulkan loader directly and expose the infallible `Entry::linked`.
 If your application can handle Vulkan being missing at runtime, you can instead enable the `loaded` feature to dynamically load Vulkan with `Entry::load`.
```

Comment:
Thanks for catching this! This text is also out of date with regard to the default, which was rolled back to `loaded`, so we should switch around the order we describe the features in and accurately identify which one is the default.

Reply:
Thanks, you're right, the text about the default should probably be corrected, too.